dev_envs/mac-env-setup.md

```diff
@@ -0,0 +1,262 @@
+# Setting up a Mac-based development environment #
+
+## Initial account setup (as administrator) ##
+
+For this section, log in as your administrator account.
+
+### Account permissions ###
+
+You’ll need to allow your standard user access to run `sudo` commands.
+First, you’ll want to take note of your account name by viewing the folders
+listed inside of the `/Users/` folder - in this example, the account
+name is **username**.
+
+```console
+$ ls /Users/
+username        Shared        fedadmin
+```
+
+Then, you’ll add your **username** to the `/etc/sudoers` file using `visudo`:
+
+```console
+$ sudo visudo
+...
+##
+# User specification
+##
+
+# root and users in group wheel can run anything on any machine as any user
+root        ALL = (ALL) ALL
+%admin      ALL = (ALL) ALL
+username    ALL = (ALL) ALL
+```
+
+Press `Esc` followed by `:wq` to save and quit `visudo`.
+
+### Install XCode command line tools ###
+
+Download XCode and accept the license agreement, then install the XCode
+command line tools.
+
+```console
+sudo xcodebuild -license
+xcode-select --install
+```
+
+### Docker ###
+
+Download the latest stable [Docker for Mac](https://www.docker.com/docker-mac).
+Install it by double-clicking the downloaded `dmg` file and dragging the
+Docker application file to the `Applications` folder.
+
+### iTerm2 (recommended) ###
+
+To set up iTerm2 - a macOS terminal replacement with some nifty features -
+visit the [iTerm2 download page](https://www.iterm2.com/downloads.html) to
+get the latest stable release. Install it by double-clicking the downloaded
+`zip` file and dragging the extracted `iTerm` application file to the
+`Applications` folder.
+
+Once installed, run the app and select the menu option
+`iTerm2 > Make iTerm2 Default Term`.
+
+NOTE: You may also `Make iTerm2 Default Term` on your standard user account.
+
+### VPN configuration ###
+
+Once you're set up with the lab (CAL) VPN connection, we recommend you set up
+split-tunneling. Instructions and a generator script are located in the
+[vpn-config-gen repository](https://github.com/cisagov/vpn-config-gen).
+
+## User environment setup ##
+
+For this section and following sections, log back in as your standard user.
+
+Note: if you installed iTerm2 above, run the iTerm2 app and select the menu
+option `iTerm2 > Make iTerm2 Default Term` to make it your default too.
+
+### Automatic package installation (recommended) ###
+
+To set up a dev environment via the [CISA `laptop` script](https://github.com/cisagov/laptop/),
+execute the following in your terminal:
+
+```console
+bash <(curl --silent https://raw.githubusercontent.com/cisagov/laptop/master/laptop)
```

Comment:
Does this need to be `/bin/bash` now as well? The manual install instructions below use `/bin/bash` instead.

Reply:
I'd also suggest using `bash` here vice `console`, given the bash-fu in the line.

Reply:
Good catch! Resolved in https://github.com/cisagov/development-guide/pull/35/commits/d7d2e3fb638a5c6e5480b00c9e427030ef000baf